usr/lib/systemd/user.conf.d/__25-defaults-SLE.conf

```diff
@@ -0,0 +1,11 @@
+# This drop-in contains specific defaults used by SUSE distributions.
+#
+# Do not edit this file, it will be overwritten on update.
+#
+# If you need to override one of the options, please create your own
+# drop-in in /etc/systemd with a higher priority (preferably greater
+# than 70).
+
+[Manager]
+# Disable default TasksMax limit (jsc#SLE-10123)
+DefaultTasksMax=infinity
```

Comment:
Isn't it redundant with usr/lib/systemd/system/user-.slice.d/25-defaults-SLE.conf ?

Reply:
* `user-.slice.d/25-defaults-SLE.conf` applies to each `user-$uid.slice`
  * path `user.slice/user-$uid.slice`
* this PR's config affects what is done to (user) units under `user@$uid.service` subtree
  * path `user.slice/user-$uid.slice/user@$uid.service/foo.unit`
  * also this is user writable thus a "voluntary" limit

So, when the former gives free-run, the latter could still limit underlying (user) unit (with upstream default).

Reply:
yeah... I mixed things up :P

Could you extend the comment so it clearly states that the limit is removed for all units spawned by the user instances ?

Thanks.